surveydoc/__main__.py

```diff
@@ -43,7 +43,8 @@ def main(credentials_path, config_path):
                 answers = recent_responses.filter(survey_results['answers']['Timestamp'], survey_results['answers'][question])
                 doc_writer.text_summary(question, answers)
 
-        document_id = doc_writer.generate_doc("{}, {}".format(subject_config['name'], datetime.now().strftime("%B %Y")))
+
```

Comment:
If you can get rid of this extra newline, I'll approve this.

Reply:
Done.